code/clue.js

```diff
@@ -1,59 +1,304 @@
+  favouriteWeapon: "poker stick"
+};
+
+const mrsPeacock = {
+  name: "Mrs Peacock",
+  color: "#378b8b",
+  description: "A successful business woman who's made some dirty deals",
+  image: "assets/peacock.png",
+  favouriteWeapon: "axe"
+};
+
+const colonelMustard = {
+  name: "Colonel Mustard",
+  color: "#d9af08",
+  description: "A colonel with a bloody past",
+  image: "assets/mustard.png",
+  favouriteWeapon: "rope"
+};
 
-// CREATE OBJECTS FOR ALL THE WEAPONS, ADD MORE CHARACTERISTICS TO THE WEAPONS IF YOU LIKE.
+const mrsGrey = {
+  name: "Mrs Grey",
+  color: "#999999",
+  description: "A housekeeper who has alot of secrets to keep",
+  image: "assets/grey.png",
+  favouriteWeapon: "poison"
+};
 
+// WEAPON OBJECTS
 const rope = {
-  name: 'rope',
-  weight: 10
-}
+  name: "Rope",
+  lastseen: "Study",
+  strength: "3",
+  color: "#dd9146",
+  image: "assets/rope.jpg",
+  id: "rope"
+};
+
+const knife = {
+  name: "Knife",
+  lastseen: "Kitchen",
+  strength: "8",
+  color: "#c18ac1",
+  image: "assets/knife.jpg",
+  id: "knife"
+};
+
+const poison = {
+  name: "Poison",
+  strength: "8",
+  lastseen: "Library",
+  color:"#669999",
+  image: "assets/poisonbottle.jpg",
+  id: "poison"
+};
+
+const axe = {
+  name: "Axe",
+  strength: "6",
+  lastseen: "Billiard room",
+  color:"#b75757",
+  image: "assets/axe.jpg",
+  id: "axe"
+};
+
+const pokerstick = {
+  name: "Poker stick",
+  strength: "6",
+  lastseen: "Living Room",
+  color:"#7676b7",
+  image: "assets/pokerstick.jpg",
+  id: "poker stick"
+};
+
+const pistol = {
+  name: "Pistol",
+  strength: "10",
+  lastseen: "Dining room",
+  color:"#3d8abd",
+  image: "assets/pistol.jpg",
+  id: "pistol"
+};
 
-// THE ROOMS ONLY HAS A NAME SO NO NEED FOR OBJECTS THERE.
 
-// NOW GROUP ALL SUSPECTS, WEAPONS AND ROOMS IN ARRAYS LIKE THIS:
+// ROOM OBJECTS
+const diningRoom = {
+  name: "Dining Room",
+  color:"#50af9c",
+  image: "assets/diningroom.jpg",
+  lastseen: "Proffesor Plum",
+  favespot: "Miss Scarlet"
+};
 
+const kitchen = {
+  name: "Kitchen",
+  color:"#e27eb0",
+  image: "assets/kitchen.jpg",
+  lastseen: "Mrs Grey",
+  favespot: "Mr Green"
+};
+
+const study = {
+  name: "Study",
+  color:"#b4b431",
+  image: "assets/study.jpg",
+  lastseen: "Mr Green",
+  favespot: "Mrs Peacock"
+};
+
+const library = {
+  name: "Library",
+  color:"#2a75c0",
+  image: "assets/library.jpg",
+  lastseen: "Miss Scarlet",
+  favespot: "Mrs Grey"
+};
+
+const billiardRoom = {
+  name: "Billiard Room",
+  color:"#d78989",
+  image: "assets/billiardroom.jpg",
+  lastseen: "Colonel Mustard",
+  favespot: "Professor Plum"
+};
+
+const livingRoom = {
+  name: "Living room",
+  color:"#e59e34",
+  image: "assets/livingroom.jpg", 
+  lastseen: "Mrs Peacock",
+  favespot: "Colonel Mustard"
+};
+
+// GROUPED SUSPECTS, WEAPONS AND ROOM OBJECTS IN ARRAYS
 const suspects = [
   mrGreen,
-  mrsWhite
-  // ...  and the rest
-]
+  professorPlum,
+  missScarlet,
+  mrsPeacock,
+  colonelMustard,
+  mrsGrey, 
```

Comment:
Actually, it's considered a good practice to add comma on the last property of object - it's because if you need to add new property, you are 100% sure that you don't have to worry about adding comma :)

Reply:
Good to know – thanks, Maks!